hub/package-manager/package/index.md

```diff
@@ -11,7 +11,20 @@ This section provides guidance about submission process for contributing package
 
 ## Independent Software Vendor (ISV) or Publisher
 
-If you are an ISV or Publisher, you can use Windows Package Manager as a distribution channel for software packages containing your applications. Windows Package Manager currently supports installers in the following formats: MSIX, MSI, and EXE.
+If you are an ISV or Publisher, you can use Windows Package Manager as a distribution channel for software packages containing your applications. Windows Package Manager currently supports installers in the following formats:  
```

Comment:
Why not just add a link to the [Supported Installer formats](https://learn.microsoft.com/en-us/windows/package-manager/winget/#supported-installer-formats) section?